source/core/types/ut_coverage_options.tps

```diff
@@ -21,13 +21,21 @@ create or replace type ut_coverage_options force as object (
   exclude_objects  ut_object_names,
   include_objects  ut_object_names,
   file_mappings    ut_file_mappings,
+  include_schema_expr varchar2(32767),
```

Comment:
Change to varchar2(4000) to keep the type SQL-compatible.
Otherwise, we cannot safely use this type in SQL